lib/widgets/new_dm_sheet.dart

```diff
@@ -415,7 +417,12 @@ class _NewDmUserListItem extends StatelessWidget {
             Avatar(userId: userId, size: 32, borderRadius: 3),
             SizedBox(width: 8),
             Expanded(
-              child: Text(store.userDisplayName(userId),
+              child: Text.rich(
+                TextSpan(
+                  children: [
+                    TextSpan(text: store.userDisplayName(userId)),
+                    UserStatusEmoji.asWidgetSpan(userId: userId, fontSize: 17,
+                      textScaler: MediaQuery.textScalerOf(context))]),
```

Comment:
nit: for list elements, give the last one a trailing comma and newline like the others, because it's not a distinguished "child" or "payload" argument; OTOH can compact this by putting the "children" argument on the first line:
```suggestion
                TextSpan(children: [
                  TextSpan(text: store.userDisplayName(userId)),
                  UserStatusEmoji.asWidgetSpan(userId: userId, fontSize: 17,
                    textScaler: MediaQuery.textScalerOf(context)),
                ]),
```